commands/uploader.go

```diff
@@ -378,6 +378,11 @@ var (
 // given "endpoint". If no state has been explicitly set, an "unknown" state
 // will be returned instead.
 func getVerifyStateFor(endpoint lfsapi.Endpoint) verifyState {
+	if v, ok := cfg.Git.Get("lfs.standalonetransferagent"); ok && v != "" {
```

Comment:
`len(v) > 0` is enough to cover this, since we will never return `("non empty string", false)`.